archetypes/Portfolio/ClaimedTokensTable/ClaimedTokenRow.tsx

```diff
@@ -71,14 +82,22 @@ export const ClaimedTokenRow: React.FC<ClaimedTokenRowProps & ClaimedRowActions>
                 >
                     Burn
                 </ActionsButton>
-                <ActionsButton
-                    size="xs"
-                    variant="primary-light"
-                    disabled={!balance.toNumber()}
-                    onClick={() => onClickCommitAction(poolAddress, side, CommitActionEnum.flip)}
+                <TooltipSelector
+                    tooltip={{
+                        key: poolIsDeprecated ? TooltipKeys.DeprecatedPoolFlipCommit : undefined,
+                    }}
```

Comment:
Could add this functionality to the pools table commit button as well. How does this behave on non deprecated pools. Does it show an empty tooltip